WalletWasabi/Helpers/EndPointParser.cs

```diff
@@ -92,9 +92,23 @@ public static bool TryParse(string endPointString, int defaultPort, out EndPoint
             string host = parts[0];
             if (host == "localhost")
             {
-                endPoint = new IPEndPoint(IPAddress.Loopback, port);
+                host = IPAddress.Loopback.ToString();
             }
-            else if (IPAddress.TryParse(host, out IPAddress addr))
+
+            bool isPortInValidRange = port < IPEndPoint.MinPort || port > IPEndPoint.MaxPort;
+            if (isPortInValidRange)
```

Comment:
This code is fine but `isPortInValidRange` doesn't really make sense as the variable name because  you are checking if the port is invalid.  Maybe `isPortInvalid`?

Reply:
Yes, thanks! https://github.com/zkSNACKs/WalletWasabi/pull/1945/commits/c39b827349e0796683cf1d61369a637d0acc3051